configure.ac

```diff
@@ -2,7 +2,10 @@
 # Process this file with autoconf to produce a configure script.
 
 AC_PREREQ([2.61])
-AC_INIT([libsass], [3.0.3], [support@moovweb.com])
+AC_INIT([libsass], m4_esyscmd_s([ echo "$LIBSASS_VERSION" | \
+                                  git describe --abbrev=4 --dirty --always --tags 2>/dev/null | \
+                                  cat VERSION 2>/dev/null | \
+                                  echo "[na]" ]), [support@moovweb.com])
```

Comment:
Looks like you need to [use m4_define](https://hg.pidgin.im/pidgin/main/file/4432aff3b90d/configure.ac#l53).